lib/modules/platform/github/index.ts

```diff
@@ -309,6 +309,7 @@ async function getBranchProtection(
   }
   const res = await githubApi.getJson<BranchProtection>(
     `repos/${config.repository}/branches/${escapeHash(branchName)}/protection`,
+    { repoCache: true },
```

Comment:
unrelated change?

Reply:
Semi-related! Can separate it though if you think that's better